### PR TITLE
Update git-xlsx-textconv.go

### PR DESCRIPTION
**Pull request by k-ta-yamada**
windowsでbuildできなかったので、修正してみました。

### CODE REVIEW

**Comment by tokuhirom** [comment]
Thans!

**Comment by k-ta-yamada** [comment]
your welcome!

File: git-xlsx-textconv.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ func main() {
 			}
 			cels := make([]string, len(row.Cells))
 			for i, cell := range row.Cells {
-				s := cell.String()
+				s, _ := cell.String()
```
Comment (claui) [inline git-xlsx-textconv.go]: @k-ta-yamada @tokuhirom What is the idea behind this change and how does this even work for you? 😳 
For me (stable Go 1.6.2 on OS X 10.10), this PR does nothing but introduce a compile error.

> […]`git-xlsx-textconv.go:30: assignment count mismatch: 2 = 1`

@k-ta-yamada Would you mind to elaborate?

**Comment by claui** [comment]
Pinging @k-ta-yamada and @tokuhirom to please have a look at [my line note](https://github.com/tokuhirom/git-xlsx-textconv/pull/4/files#r63260146). Thanks!

File: git-xlsx-textconv.go (hunk 1 above)
Comment (tokuhirom) [inline git-xlsx-textconv.go]: okay, could you send me a p-r?

**Pull request merged**